model-lab/utils/errorHandler.py

```
@@ -0,0 +1,63 @@
+def validate_image_file(file: UploadFile) -> Image.Image:
+    try:
+        
+        if not file.filename.lower().endswith(('.png', '.jpg', '.jpeg')):
+            raise HTTPException(status_code=415, detail="Unsupported image format.")
+
+       
+        image_bytes = file.file.read()                                        
+        image = Image.open(io.BytesIO(image_bytes))
+        image.verify()
+        image = Image.open(io.BytesIO(image_bytes))
+        return image
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)